flang-rt/lib/CufRuntime/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-#===-- runtime/CUDA/CMakeLists.txt -----------------------------------------===#
+#===-- lib/CufRuntime/CMakeLists.txt ---------------------------------------===#
```

Comment:
Please keep `CUDA` as it is done for the include dir.

Reply:
I still see the wrong directory name here. include is `include/flang-rt/CUDA` so like should be `lib/CUDA`